### ISSUE TITLE
Call for Ideas: Deadpop Event Ideas

### ISSUE THREAD

**Issue by Kurfursten** [d3athrow/vgstation13#32491]
Related to #32490

I would like to see if the community has any solid ideas for things to keep a lonely skeleton crew's interest up. Based on previous vine feedback, it seems like these events should avoid being busywork that makes latejoining shitty if no one participates, but present opportunities for players who are otherwise not doing much. Here are some contributions from me.

**Template**
```
**Name**:
- **Requirements**: 1-6 players
- **Announcement Text**:
- **Effect**:
```

**Name**: Self-Replicating Pizza
- **Requirements**: 1-6 players; cargo shuttle not at station
- **Announcement Text**: An anomalous pizza order has arrived on the cargo shuttle. Please process it at the crate weigher and send it on before it becomes too much of a temptation. Access to the Cargo Bay will be waived for the next five minutes.
- **Effect**: Cargo Bay and Cargo Office doors lose access requirements. The cargo shuttle immediately arrives carrying a crate with a special pizza inside. Players can choose to either to process it at the weigher and receive a cash reward for the station (higher wages next cycle) OR they can take it themselves. It can be cut endlessly to spawn extra pizza slices. After a random (high) amount of cuts, the pizza turns into a hostile mimic.

**Name**: Keep the Lights On
- **Requirements**: 1-6 players, no engineers, SMES not charging, station time is after 12:15
- **Announcement Text**: It seems your station is starting to run low on its power reserves. Central Command can send you relief power if you can get a current running in the cables. Consider running the PACMAN-type generator or Antimatter Engine. Access to Engineering will be waived for the next five minutes.
- **Effect**: Engineering doors lose requirements for five minutes. Any SME that starts charging in that time gets instantly filled up to 100% charge.

**Name**: Special Quarantine
- **Requirements**: 1-6 players, at least one mouse alive in maintenance
- **Announcement Text**: Reports from your sector regarding rodent-borne diseases are on the rise. Please cage all mice found in maintenance in the next few minutes. If you are successful, Central Command will send a cash reward and viral relief to the medbay lobby. Access requirements will be waived for five minutes and extra cages sent to the medbay lobby.
- **Effect**: Maintenance loses access for 5 minutes and mouse cages spawn in medbay lobby. If there are no uncaged mice in maintenance at the end of the timer, a cash bonus is added to the station account (higher wages next cycle) and vaccine pills with common antibodies (only) at spawned at the medbay lobby.

**Name**: Space Hollywood Research
- **Requirements**: 1-6 players, at least one dead body in morgue
- **Announcement Text**: Quentin Tarantino XII apparently has mistaken your station for a dead body storage in spite of the lack of a sign. As part of research for a new Space Hollywood film, he has asked us to submit an autopsy on one of your dead crewmen to help with a scene he's working on. Please complete an autopsy and fax us the results, and Mr. Tarantino will send you his sign of thanks. Access requirements to the morgue and internal affairs office have been waived for the next five minutes.
- **Effect**: Access lifted at morgue and IAA office. If players complete the task in time, it spawns in an unanchored superjuke, 2 bottles of champagne, and a set of sandals at the fax machine that submitted the paperwork.

**Comment by angryturnip** [comment]
I really like the idea of incentivizing power set up. I think something similar for filling the cloner with X amount of biomass and lifting medbay access restrictions would also be a good idea. Something along the lines of no doctors on the manifest and centcomm saying "oops, forgot to send medical personnel, just clone yourselves if something goes wrong." The reward could even be centcomm sending cryomix beakers or advanced trauma/burn kits.

**Comment by KingofHONK** [comment]
Something to bring dead players back into the round, especially in the cases of murderbones would be good. As it stands if you die on deadpop you may never be brought back without having to take a ghost role, and if you cant stand ghost roles then you're kinda fucked. Maybe something along the vein of a lot of our custom strike teams, like tourists, or emergency crew with access already pre loaded. Spitballing here

**Comment by Kurfursten** [comment]
I considered that but my concern is that in deadpop a lot of players will immediately log out for the night if they die and they aren't immediately dragged on the way to medbay. Spare ghosts can be rare.

**Comment by Eneocho** [comment]
**Name**: Space Vietnam Soldier.
- **Requirements**: 1-6 players, no sec
- **Announcement Text**: Sumtang Wong, a long presumed dead officer of the Space Vietnam Liberation Front has been tracked to your station. Please capture him alive and send him to the Space Vietnam Liason for recovery. He is currently highly unstable, do not make sudden moves near him. Access restrictions on maintenance will be removed temporarily until capture.
- **Effect**: Access lifted on all maintenance doors. Officer spawns and is armed with a liberator gun (almost no damage to humanoids) and is defensively hostile (attacks on sight but does not chase). On recovery station gets 5 regular grass coats, 5 rice hats and a one time payment increase as thanks for the recovery.

**Name**: Medical Mishap
- **Requirements**: 1-6 players, no medical crew
- **Announcement Text**: A clerical error accidentally deployed your station without medical staff, to alleviate this issue, please fill the cloner with at least 2000 units of biomass. Access requirements for Medical Bay and Cloning will be lifted temporarily. Supplementary medical supplies will be sent as well upon completion.
- **Effect**: Medbay and clonebay access lifted until completion. 2 advanced first aid kits, a surgeon switchtool and a mobile operation bed get teleported to medical lobby.

**Name**: Prospective Hires
- **Requirements**: 1-6 players, at least 2 ghosts
- **Announcement Text**: We have sent 2 possible employees to your station for them to learn the day to day operations. Keep them safe, they have not signed all paperwork. Return them safely after showing them the operations or give them a complete ID to properly hire them.
- **Effect**: 2 ghosts get recruited as new hires, no access cards. If returned, the station gets a one time wage increase and the ghosts go back to being ghosts. If given an ID with their name and a unique account, they get recruited and become full workers.

**Name**: Space Hollywood "Research"
- **Requirements**: 20m after Space Hollywood Research
- **Announcement Text**: Mr. Tarantino XII requests 5 feet pairs to be sent to his location; a forwarding form will be sent to coordinate the feet transport. Don't ask what he'll do with them, we honestly don't want to know either.
- **Effect**: Crew must take 5 left feet and 5 right feet on a crate to cargo, weight them, stamp the form and forward the feet. Crew gets a massive one time increase in wages (500%), 5 random rare shoes sent to the bar upon completion and a signed autograph.

**Comment by adacovsk** [comment]
remove random lotto ree

**Comment by jwhitak** [comment]
Good ideas, though I'd like to see maybe 7 to 10 minutes to go do most of those tasks

**Comment by tuypo1** [comment]
**Name**: holoboss fight
- **Requirements**: 2-8 players
- **Announcement Text**: a brief hiccup in the holodecks safeties has caused a dangerous hologram to materialize on the holodeck, please destroy it before it finds a way to escape
- **Effect**: a hostile advanced hologram NPC is spawned of some manner of somewhat dangerous enemy (such as a samurai or a lion or anything along those lines), it is confined to the holodeck and takes 5 seconds to agro on players who enter the room so corpses can be retrieved, upon being killed it drops a copy of it's weapon (or maybe one copy for each player in the room) that can be taken outside the holodeck, if 10 minutes pass without the hologram being in combat it will phase through the wall and leave into space and despawn when it leaves the Z level

Also maybe an event where a single meteor hits the station, is that already a thing I forget

**Comment by Armadingus** [comment]
**Shoal Broker Docking**:
- **Requirements**: 0 traders available, <11 players
- **Announcement Text**:  All trading ports have been opened and will remain open to make way for an official Shoal trade ship to dock with your station. All sales are final and only available for a limited time before the ship leaves, "time is money" in their own squawks.
- **Effect**: What this does is open trading ports if a player has closed them and deny all attempts to close them to not damage relations with NT. If there somehow is a player trade shuttle docked already without any traders having joined the round, it will undock by itself before a different ship will dock with one of the NPC trade brokers on it for people to interact with, it will sell trader wares and also be fluent in galactic common. Once 20 minutes have passed since it docked, all visitors will be forcibly kicked out with punchline windup box effect, before the shuttle leaves. There will be status displays inside the ship to show how long players have left until it leaves. Any profit will be split in some percentage into the player traders debit card so that latejoiners don't feel entirely cheated out of sales.

I know for a fact that people are upset about trader goods being locked away when there are no traders, this is a good solution and I think that more money sink related events would help people appreciate their money more instead of all this "oh my money is so useless why would I ever want money".

**Comment by Lumbermancer** [comment]
**La Grande Bouffe**
- **Requirements**: 1-? Players
- **Announcement Text**: The Nanotrasen Admiral is hosting a fancy banquet, but just before our chef managed to finish the final dish, they were assassinated by the Syndicate! You got to help us! We're sending you some seeds and produce. You have to make the [RANDOM NAME OF A MORE COMPLICATED DISH] and send it back to us! The Admirals reputation is at stake!
- **Effect**: Cargo, kitchin and botani lose access. You get some basic supplies on the cargo shuttle, you send the dish back on the cargo shuttle, you receive some sort of a reward and congratulatory message.

**Comment by Olb43** [comment]
Remove vines and space tumors from lowpop. No one wants to do it and it just ruins lowpop rounds sometimes to the point of having to actually call the shuttle.

**Comment by Eneocho** [comment]
Bruh, they literally just got re-enabled on lowpop yesterday. See #32490.

**Comment by Olb43** [comment]
If this gets passed then more interesting shit is happening why keep it active though. Its nothing but a chore.

**Comment by Eneocho** [comment]
The PR I linked re-enabled ALL lowpop events (like the old vendortron) including vines and (((biomass))). You can make a PR that selectively disables the vines and biomass on low pop.

**Comment by tuypo1** [comment]
from anons in the general

>Name:
Vox Metal Scrappers
>Requirements:
1-10 players, at least 1 sec and 1 ghost
>Announcement Text:
A vox metal scrapper has misidentified your seriously understaffed station as abandoned and has dispatched a team to disassemble it for the scrap. Convince them to leave before they disrupt the standard operation of <name of the station>.
>Effect:
1 to 3 ghosts get recruited as the vox metal scrappers. The metal scrappers objectives would be to stay alive and collect 3 or 4 random items (empty cans, depleted power units, broken bulbs, metal sheets, and so on).

do something to fix away missions being laggy and add a self-cloning pod to the mission if the pop is below 10 so people can just farm it for funs.
all the away missions have been neglected and forgotten and they're not even that terrible, a much better pve experience for people bored at lower populations.
and then maybe some people will get ideas for newer ones or revamped ones and it might inspire admins to use them for events again (rip mai the l4d event was one of the best lowpop events ive ever played)

**Comment by Kurfursten** [comment]
That sounds more like a midround antag idea, particularly with listed objectives

Events are a little different; they almost never require ghosts, which is a good thing because deadpop doesn't have a ton of spare ghosts

**Comment by Blithering** [comment]
Name:
"Curious case of the empty station" aka "who the fuck murderbones on deadpop"
Requirements:
1-6 players, traitor, 2-6 ghosts
Announcement text:
Spaceland yard has dispatched an investigator to solve the recent disappearances
Effect:
Sheerluck Holmes and his trusty sidekick Watsup arrive on the station to find out who's been killing everyone. If there's only one person left, it's going to be a very quick case.

**Comment by Eneocho** [comment]
`>detectives get sent to the murderboned station`
`>they get murderbones`
9/10 would make me laugh, make it happen

**Comment by Kurfursten** [comment]
I don't think that works great. First off, events don't happen all that often, maybe once every 20 minutes - and then even after it fires you have to poll for ghosts, spawn them, let them suit up, etc.. When someone deadpop murderbones, they normally don't move so slowly that you have half an hour for people to get ready. Murderman probably called the shuttle and left. Most likely people will just log out when it looks like there is no hope of revival - not wait half an hour hoping for a 3% chance this event appears.

The other problem is that on deadpop, there aren't spare ghosts to join in. So the people investigating the murder will be the murdered people. I know that as a principle we expect ghost roles to avoid acting on their ghost knowledge, but having an event especially to investigate your own death is a lot of temptation. You can argue this happens with ERTs, but 95% of ERTs are called by admins who can then answer ahelps, and they tend to be called when there are more ghost options around.

In short: I don't think it will work out how you think it will. I know you want to create a disincentive for deadpop murderbone but the events system is not it.

**Comment by adacovsk** [comment]
Trigger: no botanists
Premise: seeds have come out from the vents from the miners and into hydroponics and have spread into the hydroponics trays!
Result: randomly generated seeds are planted in free hydroponics trays

**Comment by Blithering** [comment]
Name:
Motherlode
Requirements:
Science, no miners
Announcement text: Planet cracker vessel NSS Urist has reported tractor array malfunction. Mineral rich asteroid on collision course with the station.
Effect: meteor hits station, spawns largeish blob of high value asteroid rock at point of impact, possibly pre-set vault with mining equipment scattered around as standard. Minerals delivered right to your door!

**Comment by Eneocho** [comment]
I had an interesting round where an asteroid vault "hit the station", complete with a random artifact. nt 8 disk and phazon. It was soulful.

**Comment by Kurfursten** [comment]
Neat idea though awkwardly you have to visit the roid to smelt. Hmmm. Maybe smelting equipment broke off in the collision

**Comment by Eneocho** [comment]
> Neat idea though awkwardly you have to visit the roid to smelt. Hmmm. Maybe smelting equipment broke off in the collision

You can smelt on RnD with nominal research to build the machines or "smelt" in recycling for no pay whatsoever, no need for smelting equipment.

**Comment by tuypo1** [comment]
>I know that as a principle we expect ghost roles to avoid acting on their ghost knowledge, but having an event especially to investigate your own death is a lot of temptation

My first instinct is to say it doesn't seem to cause problems with soul ramblers but then on further thought I guess that's the exact opposite problem of people avoid actually doing the investigation part of soul rambling because of their ghost knowledge, so yeah I guess the detective thing is a bad idea

**Comment by Eneocho** [comment]
> Trigger: no botanists Premise: seeds have come out from the vents from the miners and into hydroponics and have spread into the hydroponics trays! Result: randomly generated seeds are planted in free hydroponics trays

`>random giga kudzu gets planted and ends the round`
based

**Comment by Blithering** [comment]
Name:
We have the technology
Requirements: medbay staff, research staff
Announcement text: NT agent [name] has suffered extreme injuries while escaping syndicate research station with crucial data. NT infiltration protocols require all evidence be liquidated on agent's death, agent has been rerouted to [station name] for emergency treatment to prevent this failsafe triggering
Effect: stasis bag containing wounded person missing random limbs and organs is shuttled to the station, prize lucky dip of high level research disks if he is restored to full functionality (replacement limbs and organs too, no nuggets) without him dying.
